src/datastructures/Classes.java

```diff
@@ -0,0 +1,151 @@
+package datastructures;
+
+import java.util.ArrayList;
+
+public class Classes {
+	static ArrayList<Entity> entities;
+	static ArrayList<Relationship> relationships;
+
+	//*********************************************************//
+	// Constructor //
+	//*********************************************************//
+
+	public Classes() {
+		entities = new ArrayList<Entity>();
+		relationships = new ArrayList<Relationship>();
+	}
+	
+	//*********************************************************//
+	// Accessors //
+	//*********************************************************//
+	
+	public ArrayList<Entity> getEntities()
+	{
+		return entities;
+	}
+
+	public ArrayList<Relationship> getRelationships()
+	{
+		return relationships;
+	}
+	
+	//*********************************************************//
+	// Class Functions //
+	//*********************************************************//
+
+	public boolean createClass(String name) {
+		Entity e = new Entity(name);
+		if (entities.contains(e)) {
+			// Duplicate found.
+			return false;
+		}
+		return entities.add(e);
+	}
+
+	public boolean renameClass(String target, String newname) {
+		int index = entities.indexOf(new Entity(target));
+		if (index >= 0) {
+			entities.get(index).setName(newname);
+			return true;
+		} else {
+			// Target not found.
+			return false;
+		}
+	}
```

Comment:
Does this also need to check the relationships array list and change the class there as well?